pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -586,6 +607,16 @@ func (r *ReconcileBareMetalHost) actionProvisioning(prov provisioner.Provisioner
 		return actionContinueNoWrite{}
 	}
 
+	// Ensure the root device hints we're going to use are stored.
+	dirty, err := updateRootDeviceHintsForHost(info.host)
+	if err != nil {
+		return actionError{errors.Wrap(err, "Could not update root device hints")}
```

Comment:
The error is getting wrapped with the same string twice (once here and once in updateRootDeviceHintsForHost).

Reply:
fixed